webmin/os-eol-lib.pl

```diff
@@ -0,0 +1,176 @@
+# os-eol-lib.pl
+# Functions for managing OS end-of-life data
+
+use Time::Local;
+
+# eol_oses_list()
+# Returns a list of OSes for which EOL data is available
+sub eol_oses_list
+{
+return ('almalinux', 'amazon-linux', 'centos-stream', 'centos',
```

Comment:
Are these the OS names used by the EOL service we're calling?

Reply:
Yes.